cosmos/telemetry.py

```diff
@@ -47,15 +47,23 @@ def emit_usage_metrics(metrics: dict[str, object]) -> bool:
         **metrics, telemetry_version=constants.TELEMETRY_VERSION, query_string=query_string
     )
     logger.debug("Telemetry is enabled. Emitting the following usage metrics to %s: %s", telemetry_url, metrics)
-    response = httpx.get(telemetry_url, timeout=constants.TELEMETRY_TIMEOUT, follow_redirects=True)
-    if not response.is_success:
+    try:
+        response = httpx.get(telemetry_url, timeout=constants.TELEMETRY_TIMEOUT, follow_redirects=True)
+    except httpx.RequestError as e:
```

Comment:
shall we catch httpx.HTTPError to avoid task failure if any error occurs from the scarf server, too